cypress/support/fragments/inventory/inventoryInstance.js

```diff
@@ -260,6 +260,14 @@ export default {
     InventoryInstanceSelectInstanceModal.selectInstance();
     InventoryInstancesMovement.move();
   },
+  moveHoldingsToAnotherInstanceByItemTitle: (holdingName, title) => {
+    cy.do(actionsButton.click());
+    cy.do(moveHoldingsToAnotherInstanceButton.click());
+    InventoryInstanceSelectInstanceModal.waitLoading();
+    InventoryInstanceSelectInstanceModal.searchByTitle(title)
+    InventoryInstanceSelectInstanceModal.selectInstance();
+    InventoryInstancesMovement.moveFromMultiple(holdingName);
+  },
   checkAddItem:(holdingsRecrodId) => {
     cy.expect(section.find(Section({ id:holdingsRecrodId }))
       .find(Button({ id: `clickable-new-item-${holdingsRecrodId}` }))
```

Comment:
please rename this parameter from holdingsRecrodId to holdingsRecordId

Reply:
done